tests/integration/backend_test.py

```diff
@@ -662,7 +662,7 @@ def test_submission_with_group(
     shots = b.run_circuit(
         c,
         n_shots=n_shots,
-        group=os.getenv("PYTKET_REMOTE_QUANTINUUM_GROUP", default="Default - UK"),
```

Comment:
I think we should keep it here though, so that users belonging to different groups can still run the tests. Just change the default value.

Reply:
done in [9e8781b](https://github.com/CQCL/pytket-quantinuum/pull/357/commits/9e8781b7734c51e4476db5fe7633f91d16af5afc)